conf.py

```diff
@@ -16,11 +16,23 @@
 project = 'Odoo'
 copyright = 'Odoo S.A.'
 
-# `version` if the version info for the project being documented, acts as replacement for |version|,
+# `version` is the version info for the project being documented, acts as replacement for |version|,
 # also used in various other places throughout the built documents.
 # `release` is the full version, including alpha/beta/rc tags. Acts as replacement for |release|.
 version = release = '14.0'
 
+# `current_release` is the version that was last released when the current branch was frozen. It has
+# the same value as `version` on release branches (XX.0) and has the value of the release branch's
+# `version` on SaaS branches and on master.
+# `previous_release` is the release before `current_release`.
+# `next_release` is the release coming after `current_release`.
+# `latest_release` is the version that was last released when the documentation was built. Similarly
+# to `current_release`, it does not take SaaS branches into account.
+current_release = '14.0'
+previous_release = str(float(current_release) - 1.0)
+next_release = str(float(current_release) + 1.0)
+latest_release = '16.0'
```

Comment:
How does this work with intermediary releases @AntoineVDV ?
if we're reading "Odoo Online" doc on 15.2, previous release isn't 14.2 nor is next release 16.2.

Reply:
@xmo-odoo Yes, there are intermediary versions of the doc: when you're on a 15.2 database and access the documentation from your database, you end up on the `saas-15.2` branch of the doc, displayed as "Odoo Online"
https://www.odoo.com/documentation/saas-15.2/

- 15.2 is not a major release, but it still is a release.
- These placeholders are about major releases --> current wording isn't clear about that.

Reply:
I settled for `CURRENT_BRANCH` (saas-15.2), `CURRENT_VERSION` (15.2), `CURRENT_MAJOR_BRANCH` (15.0), `CURRENT_MAJOR_VERSION` (15).

Reply:
That's very clear like that 👍